sdk-python/src/test.py

```diff
@@ -0,0 +1,138 @@
+import psycopg2
+from omnilog.send_to_db import send_to_db
+import unittest
+import datetime
+
+
+class TestTypes(unittest.TestCase):
+    valid_url = "mongodb://localhost:27017/"
+    valid_log = {
+        "datetime_utc": datetime.datetime.utcnow(),
+        "input": "What is this?",
+        "output": "This is a test log",
+        "total_tokens": 5,
+    }
+
+    def test_valid_inputs(self):
+        """
+        When correct arguments are given, then connection with psycopg2 is tried and fails
```

Comment:
I'm slightly confused by this test - 

when a valid url and valid log is given, a programming error should be raised?